superset/utils/screenshots.py

```diff
@@ -159,11 +161,11 @@ def get_screenshot(
         time.sleep(SELENIUM_HEADSTART)
         try:
             logger.debug("Wait for the presence of %s", element_name)
-            element = WebDriverWait(driver, 10).until(
+            element = WebDriverWait(driver, self._element_locate_wait).until(
```

Comment:
since it is an only place where value is accessed - you can use it directly here